cmd/components.go

```diff
@@ -34,15 +34,11 @@ var ComponentsCmd = &cobra.Command{
 	Short: "List all Dapr components. Supported platforms: Kubernetes",
 	Run: func(cmd *cobra.Command, args []string) {
 		if kubernetesMode {
-			print.WarningStatusEvent(os.Stdout, "In future releases, this command will only query the \"default\" namespace by default. Please use the -n (namespace) flag, for specific namespace, or -A (all-namespaces) flag for all namespaces.")
+			print.WarningStatusEvent(os.Stdout, "In future releases, this command will only query the \"default\" namespace by default. Please use the --namespace flag, for specific namespace, or -A (--all-namespaces) flag for all namespaces.")
```

Comment:
```suggestion
			print.WarningStatusEvent(os.Stdout, "In future releases, this command will only query the \"default\" namespace by default. Please use the --namespace flag for a specific namespace, or the --all-namespaces (-A) flag for all namespaces.")
```

Should we also add a short-hand, `-n` for `--namespace`?

Reply:
Will be adding in the future ...